### ISSUE TITLE
Command "Show Build Command" fails

### ISSUE THREAD

**Issue by CurtisAMU** [microsoft/vscode-cmake-tools#2211]
### Brief Issue Summary
Running "CMake: Show Build Command" from the VS Code command palette results in an error dialog stating:
*Command 'CMake: Show Build Command' resulted in an error (Running the contributed command: 'cmake.showBuildCommand' failed.)*
![image](https://user-images.githubusercontent.com/60870607/139384824-a8f34d28-7ec6-494f-907b-5289822c5049.png)

The CMake project itself configures and builds without issues, i.e. 'CMake: Build' works fine.

### CMake Tools Diagnostics
```
{
  "os": "win32",
  "vscodeVersion": "1.61.2",
  "cmtVersion": "1.9.1",
  "configurations": [
    {
      "folder": "d:\\Projects\\Misc\\BuildSystem",
      "cmakeVersion": "3.21.3",
      "configured": true,
      "generator": "Ninja Multi-Config",
      "usesPresets": false,
      "compilers": {
        "C": "C:/Keil_v5/ARM/ARMCC/bin/armcc.exe",
        "CXX": "C:/Keil_v5/ARM/ARMCC/bin/armcc.exe"
      }
    }
  ],
  "cpptoolsIntegration": {
    "isReady": true,
    "hasCodeModel": true,
    "activeBuildType": "Debug Cortex-M4 LoMa",
    "buildTypesSeen": [
      "Debug",
      "Release",
      "RelWithDebInfo"
    ],
    "requests": [],
    "responses": [],
    "partialMatches": [],
    "targetCount": 1,
    "executablesCount": 1,
    "librariesCount": 0,
    "targets": [
      {
        "name": "AmuTarget",
        "type": "EXECUTABLE"
      }
    ]
  },
  "settings": [
    {
      "communicationMode": "automatic",
      "useCMakePresets": "auto",
      "configureOnOpen": true
    }
  ]
}
```

### Debug Log
```
[rollbar] Unhandled exception: Unhandled Promise rejection: showBuildCommand Error: Failed to get build command {}
```

**Comment by bobbrow** [comment]
We plan to release the fix for this on Monday. If you would like to verify the fix in a pre-release version of the extension, please do the following:
1. Download the VSIX here: https://github.com/microsoft/vscode-cmake-tools/suites/4455203062/artifacts/118295075
2. Unzip the VSIX file from the archive
3. Invoke the `Install from VSIX...` command in VS Code and select the VSIX file from step 2.

**Comment by CurtisAMU** [comment]
Thank you, @bobbrow, that works for me.

**Comment by bobbrow** [comment]
The fix for this has been released in 1.9.2.
